modules/acknowledgements/php/acknowledgements.class.inc

```diff
@@ -101,7 +101,7 @@ class Acknowledgements extends \NDB_Menu_Filter_Form
      * Sets up all the class variables needed for the
      * acknowledgements menu filter
      *
-     * @return true on success
+     * @return void
```

Comment:
You can't change the return type of a child class from the parent class, that's what the signature compatibility checks is checking for.

I can remove the useless return statement from these two functions, but otherwise they'll be caught when "PhanTypeMismatchReturn" is removed from the ignore list. Fixing all the mismatched returns in the codebase is beyond the scope of this PR which is only trying to strengthen the signature compatibility analysis. Adding the PhanTypeMismatchReturn check to this PR would require 269 more changes:

```
vendor/bin/phan | wc -l
     269
```

Reply:
(Fixed these 2 in the latest commit)

Reply:
@driusan Ok, I just wanted some clarification.